examples/Atlas/+atlasControllers/@InstantaneousQPController/InstantaneousQPController.m

```diff
@@ -83,7 +83,16 @@
       if isa(terrain,'DRCTerrainMap')
         terrain_map_ptr = terrain.map_handle.getPointerForMex();
       else
-        terrain_map_ptr = 0;
+        % Try to determine if we're on 64- or 32-bit matlab. From http://www.mathworks.com/matlabcentral/answers/96172#comment_270161
+        % We do this because constructQPDataPointerMex now (correctly, I think) checks that the terrain_map_ptr
+        % is actually the correct size for a pointer on this system, rather than just blindly casting from double.
+        [~, maxArraySize] = computer();
+        is64bit = maxArraySize > 2^31;
+        if is64bit
+          terrain_map_ptr = uint64(0);
+        else
+          terrain_map_ptr = uint32(0);
+        end
```

Comment:
I feel like we should never be sending pointers as doubles, even if they're zero (after all, zero-equality for doubles is a little fuzzy), so yeah, I think this should probably be used for the mex model ptr. It would probably be best to write a `getNullPointer()` method that implements this logic. 

I can do that, but probably not today.